doofinder.php

```diff
@@ -45,15 +45,15 @@ class Doofinder extends Module
 
     const GS_SHORT_DESCRIPTION = 1;
     const GS_LONG_DESCRIPTION = 2;
-    const VERSION = '4.0.6';
+    const VERSION = '4.0.7';
     const YES = 1;
     const NO = 0;
 
     public function __construct()
     {
         $this->name = 'doofinder';
         $this->tab = 'search_filter';
-        $this->version = '4.0.6';
+        $this->version = '4.0.7';
```

Comment:
why don't you use const VERSION?

Reply:
I think it is something we can change without affecting the functionality of the module. @ivan-doofinder what do you think?

Reply:
It was already tried a few months ago, but the Prestashop addons did not accept it. And surely the Prestashop validator will not accept it either. You can try it

Reply:
You are right, I just tried to validate it with this change and it does not pass the prestashop validator.